src/components/SearchForm/SearchForm.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from "react";
+import "./styles.css";
+
+const defaultValue = "";
+
+class SearchForm extends Component {
+  state = {
+    defaultValue: defaultValue,
+    onSearch: (value) => {},
+  };
+
+  constructor(props) {
+    super(props);
+    this.state.defaultValue = props.defaultValue || defaultValue;
+    this.state.onSearch = props.onSearch;
+  }
+
+  search = () => {
+    let searchInput = document.getElementById("searchInput");
+    this.state.onSearch(searchInput.value);
+  };
+
+  render() {
+    return (
+      <>
+        <div className="search-form">
```

Comment:
Your search callback doesn't call when **the user presses Enter when the input has focus**. This is a significant part of the task. Review this topic for help https://ru.reactjs.org/docs/forms.html#controlled-components

Reply:
Made SearchForm a controlled component, the input is cleared after the form submitted.
Form is submitted on having focus on input and pressing Enter
(199e1d5e57dd8e523ce5eca19af3f65fc5d22fc3)